gatsby-config.js

```diff
@@ -43,6 +43,27 @@ module.exports = {
         target: `https://account.${PORTAL_DOMAIN}`,
         secure: false, // Do not reject self-signed certificates.
         changeOrigin: true,
+        // We need to get rid of "secure" flag in the authorization cookie:
+        onProxyRes(proxyResponse, req, serverResponse) {
+          const originalCookies = proxyResponse.headers["set-cookie"];
+
+          if (Array.isArray(originalCookies)) {
+            console.log("Attempt at making the cookie insecure");
+            const newCookies = originalCookies.map((cookie) => {
+              const attributes = cookie.split("; ");
+              // Retain all cookie attributes besides "Secure" flag:
+              const filtered = attributes.filter((attr) => attr.trim().toLowerCase() !== "secure");
```

Comment:
This only ever happens in dev mode, right?

Reply:
Yup.